### DISCUSSION TITLE
E' possibile limitare la possibilità di fare segnalazioni ai soli dipendenti?

### DISCUSSION THREAD

**Discussion by DavideMor** [globaleaks/globaleaks-whistleblowing-software#3890] [Q&A] [answered]
Buongiorno, 
Ho implementato il vostro software in un server linux debian 11, ho fatto in modo che possa essere raggiunto anche dall'esterno. ( Ci siamo detti che si è solo per i dipendenti ma uno quando fa la segnalazione se è in ufficio con altri colleghi?).
Però per evitare che dall'esterno facciano segnalazioni anche gente con autorizzata volevamo inserire un codice che consegnamo con l'informativa per identificare i soggetti autorizzati ad aprire una segnalazione( dipendenti). Come faccio però dal software a verificare il codice che verrà inserito in una casella di testo prima di potergli far rispondere al questionario e creare la segnalazione.

Grazie

**Comment by evilaliv3** [comment, marked as answer]
Grazie per la domanda.

Non prevediamo una tale possibilità e non la riteniamo opportuna nel contesto d'uso che prevede un pubblico di possibili segnalanti molto piu allargato dei soli dipendenti, includente i fornitori, gli azionisti, i tirocinanti, i possibili o ex appartenenti a queste categorie, ecc.

Al contrario facendo un questionario ben fatto si riesce a limitare le segnalazioni a quelle di contesto;
Ad esempio limitando il contesto soggettivo con una domanda (Che rapporto hai con la società?) ed il contesto oggettivo con una domanda (Che tipo di situazione vuoi segnalare)  che abbia le precise opzioni di legge.
